pom.xml

```diff
@@ -200,6 +206,13 @@
         </dependency>
     </dependencies>
 
+    <repositories>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
```

Comment:
We shouldn't use jitpack for transitive dependencies, even if they are optional. The artifact needs to be published/permanent.

Reply:
They apparently publish their newer versions only on jitpack. 
https://github.com/everit-org/json-schema/blob/master/README.md#maven-installation

Reply:
Can users still edit and unpublish Jitpack artifacts, or is this version frozen ?

Reply:
From "https://jitpack.io/docs/API/"

You need to be authenticated when deleting a build and your user needs to have push permissions to the git repository. Note that you can only delete failed builds, snapshot builds or tag builds newer than 7 days. Older versions can’t be deleted because there could be builds depending on those versions.